### ISSUE TITLE
suffix and other parameters are silently ignored when providing metadata

### ISSUE THREAD

**Issue by nearwood** [google-github-actions/deploy-cloudrun#386]
### TL;DR

When providing metadata, I get a warning saying [some values will be ignored](https://github.com/google-github-actions/deploy-cloudrun/blob/v0.10.1/src/main.ts#L141), but [other values are too](https://github.com/google-github-actions/deploy-cloudrun/blob/v0.10.1/src/main.ts#L170-L175).

### Expected behavior

I wanted to use suffix to provide a custom revision name suffix.

### Observed behavior

I got the metadata warning (I'm not sure why, I don't explicitly specificy the values it checks in the if statement): `Metadata YAML provided, ignoring: "image", "service", "env_vars", "secrets", and "timeout"`.

Suffix was silently ignored, so I looked at the action source to determine why. It also appears that timeout is part of the warning but isn't actually ignored, since it's outside the metadata if statement.

### Action YAML

```yaml
- name: Deploy
      id: 'deploy'
      uses: 'google-github-actions/deploy-cloudrun@v0'
      with:
        project_id: "${{ env.CLOUDRUN_PROJECT_ID }}"
        region: "${{ env.GC_REGION }}"
        metadata: packages/documents-app/config/service.yaml
        suffix: ${{ github.run_number }}-${{ github.run_attempt }}
```

### Log output

```text
Run google-github-actions/deploy-cloudrun@v0
  with:
    project_id: myproject-next
    region: us-central1
    metadata: packages/documents-app/config/service.yaml
    suffix: 33-1
    no_traffic: false
  env:
    CLOUDRUN_PROJECT_ID: myproject-next
    GC_REGION: us-central1
    REPOSITORY: my-platform-next
    IMAGE: documents-app
    CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE: /home/runner/work/my-platform/my-platform/gha-creds-[2](https://github.com/org/my-platform/actions/runs/3054555416/jobs/4926593792#step:8:2)7d66cca722f[3](https://github.com/org/my-platform/actions/runs/3054555416/jobs/4926593792#step:8:3)7c1.json
    GOOGLE_APPLICATION_CREDENTIALS: /home/runner/work/my-platform/my-platform/gha-creds-27d[6](https://github.com/org/my-platform/actions/runs/3054555416/jobs/4926593792#step:8:6)6cca[7](https://github.com/org/my-platform/actions/runs/3054555416/jobs/4926593792#step:8:7)[22](https://github.com/org/my-platform/actions/runs/3054555416/jobs/4926593792#step:8:23)f37c1.json
    GOOGLE_GHA_CREDS_PATH: /home/runner/work/my-platform/my-platform/gha-creds-27d66cca722f37c1.json
    CLOUDSDK_CORE_PROJECT: org-myproject-shared
    CLOUDSDK_PROJECT: org-myproject-shared
    GCLOUD_PROJECT: org-myproject-shared
    GCP_PROJECT: org-myproject-shared
    GOOGLE_CLOUD_PROJECT: org-myproject-shared
    CLOUDSDK_METRICS_ENVIRONMENT: github-actions-setup-gcloud
Warning: Metadata YAML provided, ignoring: "image", "service", "env_vars", "secrets", and "timeout" inputs.
Running: gcloud beta run services replace packages/documents-app/config/service.yaml --platform managed --region us-central1 --project myproject-next --format json
```
